scripts/dowhop-description-form-create.js

```diff
@@ -25,6 +26,7 @@ function submitNewDoWhopEntry(e) {
       titleDescription.value,
       // whyDescription.value,
```

Comment:
Thanks for noticing this! We should be validating whyDescription field and it it should NOT have been commented out. Please un-comment and fix the typo for 'snap' that you found.  👍